draft-ietf-lake-edhoc.md

```diff
@@ -547,14 +547,12 @@ See {{COSE}} for more examples.
 
 #### Compact Encoding of ID_CRED for 'kid' {#compact-kid}
 
-To comply with the EDHOC message size requirements, two optimizations are made for the case when ID_CRED_x contains a single 'kid' parameter.
+To comply with the LAKE message overhead requirements, see {{draft-ietf-lake-reqs}}, two optimizations are made for the case when ID_CRED_x contains a single 'kid' parameter.
```

Comment:
overhead seems worse than size. Overhead makes more sense for OSCORE when you have a message to send.

Reply:
@emanjon Do you prefer "size"? I'm neutral.